packages/microsoft-reactnative-sampleapps/windows/SampleAppCS/SampleAppCS.csproj

```diff
@@ -77,7 +77,7 @@
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <ErrorReport>prompt</ErrorReport>
     <Prefer32Bit>true</Prefer32Bit>
-    <UseDotNetNativeToolchain>true</UseDotNetNativeToolchain>
+    <UseDotNetNativeToolchain>false</UseDotNetNativeToolchain>
```

Comment:
Can we do this only in arm64 (assume that that was the issue?)
Dont we need to use .net native to be able to submit to the store?  -- And therefore we should ensure we are able to test that?

Reply:
Currently the .Net Native fails for all platforms. It is very high on my list to fix soon.

Reply:
Also, please note that this is Debug configuration and it must be false by default. It is usually only true for Release.

---
In reply to: [403306080](https://github.com/microsoft/react-native-windows/pull/4491#discussion_r403306080) [](ancestors = 403306080)

Reply:
this is for debug. it's enabled for release.